src/containers/applications/tabs2.scss

```diff
@@ -54,27 +54,26 @@ rediv {
   display: flex;
   flex-direction: column;
   align-items: flex-end;
-  // justify-content: flex-end;
   padding: 0 32px;
-  color: #333;
+  color: var(--dark-txt);
 }
 
 .eqCont {
   font-size: 0.22em;
-  color: #888;
+  color: var(--dark-txt);
 }
 
 .msrVal {
   display: flex;
   padding: 16px 0;
   font-size: 0.64em;
-  color: #888;
+  color: var(--dark-txt);
```

Comment:
leave it as #888

Reply:
cant do that (wont work in dark mode) can I use --sat-txt instead its pretty close ?

Reply:
yes

Reply:
dn